zgrab_schema.py

```diff
@@ -800,29 +805,23 @@
                     "compression":String(),
                 }),
             }),
-            "dh_key_exchange": SubRecord({
-                "params": SubRecord({
-                    "prime": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "generator": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
-                    "client_private": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
-                    }),
+            "key_exchange": SubRecord({
+                "ed25519sha256_params": SubRecord({
+                    "server_public": Binary(),
+                }),
+                "ecdh_params": SubRecord({
                     "server_public": SubRecord({
-                        "value":Binary(),
-                        "length":Integer()
+                        "x": golang_crypto_param,
+                        "y": golang_crypto_param,
                     }),
                 }),
+                "dh_params": SubRecord({
+                    "prime": golang_crypto_param,
+                    "generator": golang_crypto_param,
+                    "client_public": golang_crypto_param,
```

Comment:
[The code that produces](https://github.com/zmap/zgrab/blob/master/ztools/keys/dhe.go#L45) this output is shared between the XSSH and TLS grabbers. Modifying it would affect TLS output, too. I'm not sure whether that's desired.